tools/droplets/fabric/scripts/01-initial-setup

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Scripts in this directory are run during the build process.
+# each script will be uploaded to /tmp on your build droplet, 
+# given execute permissions and run.  The cleanup process will
+# remove the scripts from your build system after they have run
+# if you use the build_image task.
+#
+echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends && \
+    echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends && \
+    apt-get -q update && \
+    apt-get -q install locales && \
+    locale-gen en_US.UTF-8 && \
+    apt-get -q dist-upgrade -y
+wget https://www.zulip.org/dist/releases/zulip-server-latest.tar.gz
+tar -xf zulip-server-latest.tar.gz
+./zulip-server-*/scripts/setup/install --email=mail@example.com --hostname $(hostname -f) --no-init-db --self-signed-cert
```

Comment:
We should remove various files from `/etc/zulip` after this step to delete secrets.

`rm -f /etc/zulip/zulip-secrets.conf /etc/zulip/settings.py` is the code for that (borrowed from `docker-zulip`)